### ISSUE TITLE
Kineticist feat suggestion

### ISSUE THREAD

**Issue by Mythalar** [Truinto/DarkCodex#25]
Hey there !

Love your modd, it allows builds I'm very satisfied with :)

I have two suggestions and a question :

- Would you mind adding a feat specific to the Psychokineticist that allows to overcome the skill penalties that are tied to burns?
There were some mods for Kingmaker that gave an untyped bonus equal to the penalty to do this (coupled with a reduction of the will save penalty, -1 instead of -2, for Call of the Wilds IIRC but as you wish ^^ )

- Would it be possible to make it so Everlasting Judgement mythic ability also affect the Judge's Judgement Aura?

- How difficult would it be to change an archetype main stat?
IE I love the Psychokineticist (who would have guessed...) but rather have it CHA based ?

Sorry for the approximate English and thank you !

**Comment by Truinto** [comment]
Yeah, I had a feature in Fumi's Codex that reduced these penalties. Will be no issue to recreate this feature.
Same for the other requests.

Will do when I have time.

**Comment by Truinto** [comment]
hehe

**Comment by Mythalar** [comment]
Oh that's so nice 'o'

I'll test judgement aura thanks !

**Comment by Mythalar** [comment]
Little update !
Everlasting judgement works perfectly but there is a surprising little bug with the main stat change of psychokineticist :
when i create a new character or load a save the DC of susbstance infusions does not initially take into account that racial stat bonus.
IE I made an elfe psycho (switched to INT main stat) with 19 (17+2 racial) INT
The DC of my substance infused blast is 13, but should be 14

The stranger point yet :
If I use Toy Box to change and revert back immediatly the INT score, the DC is updated to 14
Same if I put on an INT head gear -> updated to 15 (13 base + 1 racial + 1 headgear)
But if I save my game whith the headband and load the DC is back to 14 (13 base + 1 head)

Well, actually not much of an issue since just changing my INT score updates the DC, but quite surprising ^^
